### PR TITLE
refactor(components): [virtual-list, visual-hidden] use useNamespace

### PR DESCRIPTION
**Pull request by gjfei**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/virtual-list/src/components/scrollbar.ts [hunk 1]
```diff
@@ -27,6 +28,8 @@ const ScrollBar = defineComponent({
   setup(props, { emit }) {
     const GAP = 4 // top 2 + bottom 2 | left 2 + right 2
 
+    const nsVirtualScrollbar = useNamespace('virtual-scrollbar')
+    const nsScrollThumb = useNamespace('scrollbar__thumb')
```
Comment (JeremyWuuuuu) [inline packages/components/virtual-list/src/components/scrollbar.ts:32]:
```suggestion
    const nsScrollThumb = useNamespace('scrollbar')
```

File: packages/components/virtual-list/src/components/scrollbar.ts [hunk 2]
```diff
@@ -260,15 +263,15 @@ const ScrollBar = defineComponent({
         {
           role: 'presentation',
           ref: trackRef,
-          class: 'el-virtual-scrollbar',
+          class: nsVirtualScrollbar.b(),
```
Comment (JeremyWuuuuu) [inline packages/components/virtual-list/src/components/scrollbar.ts:266]:
```suggestion
          class: nsVirtualScrollbar.e('thumb'),
```

File: packages/components/virtual-list/src/components/scrollbar.ts [hunk 3]
```diff
@@ -260,15 +263,15 @@ const ScrollBar = defineComponent({
...
           style: trackStyle.value,
           onMousedown: withModifiers(clickTrackHandler, ['stop', 'prevent']),
         },
         h(
           'div',
           {
             ref: thumbRef,
-            class: 'el-scrollbar__thumb',
+            class: nsScrollThumb.b(),
```
Comment (JeremyWuuuuu) [inline packages/components/virtual-list/src/components/scrollbar.ts:274]:
```suggestion
            class: nsVirtualScrollbar.e('thumb'),
```

**Review by jw-foss — COMMENTED** [review]
Left a few comments for you 🏓

File: packages/components/virtual-list/src/components/scrollbar.ts (hunk 2 above)
Comment (gjfei) [reply packages/components/virtual-list/src/components/scrollbar.ts:266]: 这里我不是很明白

**Comment by gjfei** [comment]
我不小心删除了我的分支

File: packages/components/virtual-list/src/components/scrollbar.ts (hunk 1 above)
Comment (sxzz) [reply packages/components/virtual-list/src/components/scrollbar.ts:32]: const nsScrollbar = useNamespace('scrollbar')

**Review by iamkun — APPROVED** [review]

**Pull request merged**